### PR TITLE
fix(nft-sell-orders): makerRelayerFees fixed

### PR DESCRIPTION
**Pull request by jmichel-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/index.ts [hunk 1]
```diff
@@ -21,6 +21,9 @@ import {
 
 export const cancelNftListings = async (asset: NftAsset, signer: Signer) => {
   // TODO: on front end maybe worth having a way for users to select which one of their sell orders they want to cancel and then input the order directly to this function.
+  if (!asset.sell_orders) {
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/payment/nfts/index.ts:24]: I think sell_orders is an array, so you may want to check !asset.sell_orders.length

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts [hunk 2]
```diff
@@ -1072,7 +1076,7 @@ export async function _makeSellOrder({
     basePrice: new BigNumber(basePrice.toString()),
     calldata,
     englishAuctionReservePrice: reservePrice ? new BigNumber(reservePrice.toString()) : undefined,
-    exchange: '0x7Be8076f4EA4A4AD08075C2508e481d6C946D12b',
+    exchange: '0x7Be8076f4EA4A4AD08075C2508e481d6C946D12b'.toLowerCase(),
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts:1079]: Can you add a comment here since this isn't usual practice

**Review by plondon — CHANGES_REQUESTED** [review]
small things

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts (hunk 2 above)
Comment (jmichel-bc) [reply packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts:1079]: I will update it to be hardcoded to lower case instead, my bad for the hacky fix!

Comment (plondon) [reply packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts:1079]: All good!

File: packages/blockchain-wallet-v4/src/redux/payment/nfts/index.ts (hunk 1 above)
Comment (Donone19) [reply packages/blockchain-wallet-v4/src/redux/payment/nfts/index.ts:24]: Thanks.

**Pull request merged**